packages/web-components/src/kumo/button/button.options.ts

```diff
@@ -0,0 +1,32 @@
+import type { ValuesOf } from '../../utils/index.js';
+export { ButtonType, ButtonOptions, ButtonFormTarget } from '../../button/button.options.js';
+
+/**
+ * ButtonAppearance constants
+ * @public
+ */
+export const ButtonAppearance = {
```

Comment:
Wondering about naming we can figure out later but will this conflict with ButtonAppearance from fluent when we have the new package?

Reply:
Let me go through and disambiguate these option values by prefixing with `Kumo-`

Reply:
Disambiguated!